berserk/clients/teams.py

```diff
@@ -73,3 +73,19 @@ def accept_join_request(self, team_id: str, user_id: str) -> None:
         """
         path = f"/api/team/{team_id}/request/{user_id}/accept"
         self._r.post(path)
+
+    def get_team(self, team_id: str) -> Dict[str, Any]:
+        """Get the information about the team
+
+        :return: the information about the team
+        """
+        path = f"/api/team/{team_id}"
+        return self._r.get(path)
```

Comment:
You may wan't to check for inexistent teams.

It will return something like this:

```json
{
  "error": "Not found"
}
```

Maybe we wan't to raise something in that case ?

Reply:
It probably returns a 404 status code which raises an exception already.